### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by ascherer**
After the first two large commits, I try to add smaller changes. :innocent:

### CODE REVIEW

**Comment by gdr-at-ms** [comment]
Squash and clean up description.

**Comment by ascherer** [comment]
Done.

File: CppCoreGuidelines.md [hunk 1]
```diff
@@ -5063,7 +5063,7 @@ Of course there are ways of making `==` work in a hierarchy, but the naive appro
 
 ## <a name="SS-containers"></a> C.con: Containers and other resource handles
 
-A container is an object holding a sequence of objects of some type; `std::vector` is the archetypical container.
+A container is an object holding a sequence of objects of some type; `std::vector` is the archetypal container.
 A resource handle is a class that owns a resource; `std::vector` is the typical resource handle; its resource is its sequence of elements.
```
Comment (gdr-at-ms) [inline CppCoreGuidelines.md]: "archetypical" is fine: http://www.thesaurus.com/browse/archetypical

**Comment by ascherer** [comment]
Thanks for the hint to the thesaurus. I unfixed the adjective and improved the fix for 'in excess'.

**Pull request merged**